src/util/Stage.java

```diff
@@ -0,0 +1,48 @@
+package util;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class Stage
+{
+    private Stage instance;
+    private JButton Start;
```

Comment:
I think there is no need to store the button in an attribute of this class since it is only created once and never accessed again in this class